kyaml/fieldmeta/fieldmeta.go

```diff
@@ -166,3 +217,20 @@ func (it FieldValueType) TagForValue(value string) string {
 	}
 	return ""
 }
+
+const (
+	// CLIDefinitionsPrefix is the prefix for cli definition keys.
+	CLIDefinitionsPrefix = "io.k8s.cli."
+
+	// SetterDefinitionPrefix is the prefix for setter definition keys.
+	SetterDefinitionPrefix = CLIDefinitionsPrefix + "setters."
+
+	// SubstitutionDefinitionPrefix is the prefix for substitution definition keys.
+	SubstitutionDefinitionPrefix = CLIDefinitionsPrefix + "substitutions."
+
+	// DefinitionsPrefix is the prefix used to reference definitions in the OpenAPI
+	DefinitionsPrefix = "#/definitions/"
+
+	// ShortHandRef is the shorthand reference to setters and substitutions
+	ShortHandRef = "$openAPI"
```

Comment:
We need to decide on the ShortHandRef name

Reply:
How about `kpt-set` to tie this back into the concept of a `set` in KPT?

Reply:
@frankfarzan Updated PR with getter and setter for ShortHandRef. However, this ref should be set in upstream kpt repo as there shouldn't be any kpt references in kustomize. Will make that change once this is merged.